recipe/run_test.py

```
@@ -5,3 +5,19 @@
     print(ogr.GetDriver(i).GetName())
 
 import os1_hw
+
+def has_geos():
+    pnt1 = ogr.CreateGeometryFromWkt( 'POINT(10 20)' )
+    pnt2 = ogr.CreateGeometryFromWkt( 'POINT(30 20)' )
+    ogrex = ogr.GetUseExceptions()
+    ogr.DontUseExceptions()
+    hasgeos = pnt1.Union( pnt2 ) is not None
+    if ogrex:
+        ogr.UseExceptions()
+    return hasgeos
+
+if has_geos():
+  print('GDAL has GEOS compiled in')
```

:+1: Let's just raise an error if geos support is not there.